app/components/containerInspect/containerInspectController.js

```diff
@@ -0,0 +1,18 @@
+angular.module('containerInspect', [])
+.controller('ContainerInspectController', ['$scope', '$transition$', 'Notifications', 'ContainerService',
+function ($scope, $transition$, Notifications, ContainerService) {
+  $scope.state = {};
+  $scope.state.loaded = false;
+  
+  ContainerService.inspect($transition$.params().id)
+  .then(function success(d) {
+    $scope.data = d;
```

Comment:
I believe the associated scope object should have a name instead of simply `data`. I think it's clearer when you check the related HTML code. Something like `$scope.containerInfo`.